module-types/Cisco/C6800-SUP6T.yaml

```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Cisco
+model: C6800-SUP6T-XL
+part_number: C6800-SUP6T-XL
+comments: '[Cisco Catalyst 6800 Series Supervisor Engine Data Sheet](https://www.cisco.com/c/en/us/products/collateral/interfaces-modules/catalyst-6800-series-supervisor-engine-6t/datasheet-c78-736408.html)'
+console-ports:
+  - name: con 0
+    type: rj-45
+  - name: usb 0
+    type: other
+  - name: usb 1
+    type: other
```

Comment:
This can cause a naming collision with more then one sup in the chassis.

Distinguish them somehow by adding {module} into the mix

Reply:
Fixed